msal/application.py

```diff
@@ -795,7 +835,10 @@ def acquire_token_by_device_flow(self, flow, **kwargs):
         """
         return self.client.obtain_token_by_device_flow(
             flow,
-            data=dict(kwargs.pop("data", {}), code=flow["device_code"]),
+            data=dict(
+                kwargs.pop("data", {}),
+                code=flow["device_code"],
+                claims=_merge_claims_and_capabilities(self._client_capabilities, claims)),
                 # 2018-10-4 Hack:
                 # during transition period,
                 # service seemingly need both device_code and code parameter.
```

Comment:
My bad. Did not realize the previous change split this comment with its corresponding code
```suggestion
                code=flow["device_code"],  # 2018-10-4 Hack:
                    # during transition period,
                    # service seemingly need both device_code and code parameter.
                claims=_merge_claims_and_capabilities(self._client_capabilities, claims),
                ),
```